src/main/java/seedu/address/model/property/Property.java

```diff
@@ -124,7 +135,11 @@ public String toString() {
                 .append("; Description: ")
                 .append(getDescription())
                 .append("; Seller: ")
-                .append(getSeller());
+                .append(getSeller())
+                .append("; Characteristics: ")
+                .append(getCharacteristics()
+                        .map(Characteristics::toString)
+                        .orElse("Not Specified"));
```

Comment:
The order we defined characteristics initially is behind the tags prefix, but the toString methods will display the characteristics before the tags. Maybe it will be more consistent to move the appending of characteristics to after the tags?

Reply:
Right I've changed it thanks!